### DISCUSSION TITLE
Support for RTS3903N Yi based cameras

### DISCUSSION THREAD

**Discussion by dostick** [roleoroleo/yi-hack-Allwinner-v2#966] [Ideas]
Hello,
I've tested and succesfully used this [RTSP server](https://github.com/cjj25/Yi-RTS3903N-RTSPServer) by @cjj25 with my RTS3903N based camera, brand CACAGOO, model XY-R9820-F4, prefix home_r10m, firmware 7.1.00.25A_202002271051.

I'd like to ask you if there's a way to include this cameras and firmware in your project or exporting some of the features (specially PTZ 🥹 ) to the project I've mentioned before.

**Comment by roleoroleo** [comment]
This platform is not compatible with my current projects (MStar and Allwinner) and I don't own this type of cam.
Sorry but I can't work on it.

**Reply by dostick** [reply]
I understand. Thank you for your fast answer.
